packages/contracts/contracts/core/HackMoneyVault.sol

```diff
@@ -151,6 +156,7 @@ contract HackMoneyVault is Multicall, Ownable, BaseVault {
     }
 
     // helper to set strategy size
+    // TODO: Do we need this?
```

Comment:
do you need it for the UI? Could also need it for trading to know how much we still can trade

Reply:
We can get it from subgraph easily